recipe/meta.yaml

```diff
@@ -16,13 +16,14 @@ build:
 
 requirements:
   host:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - pip
   run:
-    - python >=3.7,<3.10
+    - python >=3.10,<3.13
     - setuptools
-    - openff-toolkit >=0.10.6, <0.11.0
-    - smirnoff-plugins >=0.0.3
+    - openff-toolkit >=0.11
+    - openff-interchange-base >=0.3.25
```

Comment:
All the way down to 0.3.0 probably works, but the charge caching but is present in most versions 0.3.18 and older, and there are some bugfixes related to virtual sites around this version